Client/src/Common/Submitter/ISubmitterService.cs

```diff
@@ -37,6 +37,25 @@ public interface ISubmitterService : IDisposable
   /// </summary>
   string SessionId { get; }
 
+  /// <summary>
+  ///   The method submit will execute task asynchronously on the server and will serialize object[] for Service method
+  ///   MethodName(Object[] arguments)
+  /// </summary>
+  /// <param name="methodName">The name of the method inside the service</param>
+  /// <param name="arguments">A list of object that can be passed in parameters of the function</param>
+  /// <param name="handler">The handler callBack implemented as IServiceInvocationHandler to get response or result or error</param>
+  /// <param name="maxRetries">The number of retry before fail to submit task. Default = 5 retries</param>
+  /// <param name="taskOptions">
+  ///   TaskOptions argument to override default taskOptions in Session.
+  ///   If non null it will override the default taskOptions in SessionService for client or given by taskHandler for worker
+  /// </param>
+  /// <returns>Return the taskId string</returns>
+  string Submit(string                    methodName,
+                object[]                  arguments,
```

Comment:
This was voluntarily removed and replaced by the method extensions (hence the includes in the other files)